package.json

```diff
@@ -12,10 +12,10 @@
     "build": "tsc"
   },
   "dependencies": {
-    "@biotope/generator": "^2.6.0",
+    "@biotope/generator": "^2",
     "commander": "^2.15.1",
     "git-clone": "^0.1.0",
-    "npm": "^6.4.1",
+    "npm": "^6.9.0",
```

Comment:
Q: Do we need this dep? Is it not enough to have `engines` in `package.json`?

Reply:
Mhmm good question :) I don't know let's try remove it and see what happens 👍

Reply:
@tiagomapmarques @jurekbarth this is a dependency, as we run npm by node and not by executing the cli

Reply:
@biotope/core-enforcer second approval needed for merge